crates/networking/rpc/engine/fork_choice.rs

```diff
@@ -19,24 +20,18 @@ use crate::{
 #[derive(Debug)]
 pub struct ForkChoiceUpdatedV3 {
     pub fork_choice_state: ForkChoiceState,
-    #[allow(unused)]
-    pub payload_attributes: Result<Option<PayloadAttributesV3>, String>,
```

Comment:
I changed this, since the object type should reflect the json schema that we receive via RPC. Also this was leaking to the outside (see rpc/engine/forkchoice)